lib/ar_lazy_preload/active_record/collection_proxy.rb

```diff
@@ -3,7 +3,7 @@
 module ArLazyPreload
   # ActiveRecord::CollectionProxy patch for getting records for preload
   module CollectionProxy
-    private
+    protected
```

Comment:
Why changed to `protected`?

Reply:
I only change `private` to `protected` if it's accessed within the same class like:
```ruby
class SomePOROClass
  def initialize(data_hash)
     @data_hash = data_hash
  end

  def merge(hash_or_entry)
    new_hash = if hash_or_entry.is_a?(SomePOROClass)
      data_hash.merge(hash_or_entry.data_hash))
    else
      data_hash.merge(hash_or_entry)
    end
  end
  
  protected

  attr_reader :data_hash
end
```

Reply:
okay, changed back to private